toolkit/tools/imagegen/configuration/user.go

```diff
@@ -87,6 +91,16 @@ func (p *User) UIDIsValid() (err error) {
 	return
 }
 
+// This function does not account for the Password key being intentionally
```

Comment:
nit - reverse the order of comments here. Put `// PasswordIsValid` comment first

Reply:
done :)